import_3dm.py

```diff
@@ -1,3 +1,15 @@
+bl_info = {
+    "name": "Import Rhinoceros 3D",
+    "author": "jesterKing",
+    "version": (0, 0, 1),
+    "blender": (2, 80, 0),
+    "location": "File > Import > Rhinoceros 3D (.3dm)",
+    "description": "This Addon lets you import all render meshes from Rhinoceros 3dm files",
+    "warning": "",
```

Comment:
We can tell here that not all data is yet supported

```suggestion
    "warning": "The importer doesn't handle all data in 3dm files yet",
```